From 3bf2e8b95d5a2c8000a7e46b73f0b92c5319e927 Mon Sep 17 00:00:00 2001
From: Christophe Fergeau <cfergeau@mandriva.com>
Date: Sat, 10 Oct 2009 12:16:35 +0000
Subject: link daap plugin with RHYTHMBOX_LIBS (bug #597899)

The DAAP plugin uses libsoup, so it needs to link with libsoup which is
part of RHYTHMBOX_LIBS.
---
diff --git a/plugins/daap/Makefile.am b/plugins/daap/Makefile.am
index 90935ee..dfc1058 100644
--- a/plugins/daap/Makefile.am
+++ b/plugins/daap/Makefile.am
@@ -35,6 +35,7 @@ libdaap_la_LIBTOOLFLAGS = --tag=disable-static
 libdaap_la_LIBADD = 					\
 	$(top_builddir)/shell/librhythmbox-core.la	\
 	$(DBUS_LIBS)					\
+	$(RHYTHMBOX_LIBS)				\
 	$(MDNS_LIBS)
 
 INCLUDES = 						\
--
cgit v0.8.2
